ds/ds.go

```diff
@@ -236,12 +236,12 @@ func genSortTxt(key string, operator byte) lessFunc {
 				return false
 			}
 		}
-		n1, err := strconv.ParseFloat(c1.Text[key], 10)
+		n1, err := strconv.ParseFloat(c1.Text[key], 64)
```

Comment:
why make this 64? I assume the author had a reason to limit to 10 bits? Same question applies to all parsefloat changes.

Reply:
we can make it 32, but this change really have no effect, it will still run atof64 like with 10 that we have right now
```go
func ParseFloat(s string, bitSize int) (float64, error) {
	f, n, err := parseFloatPrefix(s, bitSize)
	if n != len(s) && (err == nil || err.(*NumError).Err != ErrSyntax) {
		return 0, syntaxError(fnParseFloat, s)
	}
	return f, err
}

SA1030 - Invalid argument in call to a strconv function
This check validates the format, number base and bit size arguments of the various parsing and formatting functions in strconv.

func parseFloatPrefix(s string, bitSize int) (float64, int, error) {
	if bitSize == 32 {
		f, n, err := atof32(s)
		return float64(f), n, err
	}
	return atof64(s)
}

```

Reply:
I'm still not understanding. The bitSize argument is not ignored -- it is used in the call to parseFloatPrefix. The author set it to 10 in this call. What is the reason to change it to 64?

Reply:
This is not how ParseFloat works, if bitSize is 32 it will parse to float32 and return float64 of a result, otherwise it will always use atof64. See bitSize only used in 

```
if bitSize == 32  { ...
}
```

So if we set bitSize to 10, -10 or 25 it will have the same effect as 64, but it confusing for the reader.

Reply:
https://pkg.go.dev/strconv#ParseFloat 10 is equivalent to 64, as it is the default. But 64 really does make more sense.